js/index.js

```diff
@@ -6,6 +6,8 @@ const siteContent = {
     "nav-item-4": "Features",
     "nav-item-5": "About",
     "nav-item-6": "Contact",
+    "nav-item-7": "Test",
+    "nav-item-8": "DOMinate",
```

Comment:
I like this method of adding the new texts. It's very scaleable under the right circumstances (ie. if you were using a loop to inject the anchor elements!). 👍